src/components/tile/responsive-tile-container/responsive-tile-container.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { GAP_VALUES, Gap, getGapValue } from "style/utils/box-gap";
+import { mount } from "enzyme";
+import ResponsiveTileContainer from "./responsive-tile-container.component";
+import { assertStyleMatch } from "../../../__spec_helper__/test-utils";
+
+describe("ResponsiveTileContainer", () => {
+  it.each<Gap>([...GAP_VALUES, "20px", "20%"])(
+    "has styles applied when column-gap is set to %s",
+    (gapValue) => {
+      const wrapper = mount(
+        <ResponsiveTileContainer columnGap={gapValue}>
+          content
+        </ResponsiveTileContainer>
+      );
+      const columnGap = getGapValue(gapValue);
+      assertStyleMatch({ columnGap }, wrapper);
+    }
+  );
+
+  it("does not render when no children are passed", () => {
+    render(<ResponsiveTileContainer data-testid="responsive-tile-container" />);
+
+    expect(
+      screen.queryByTestId("responsive-tile-container")
+    ).not.toBeInTheDocument();
+  });
+
+  it("does not render when falsy children are passed", () => {
```

Comment:
comment: if we make `children` required we can remove this test

Reply:
Children is a ReactNode and can be null for example.
I have this test to cover 
```
if (!children) {
    return <></>;
  }
```